src/image_artifact.js

```diff
@@ -52,10 +52,12 @@ exports.getUploader = function (artifactUploader, containerEngineName = "docker"
  * Eg. image `foo:latest` packaged as `foo_latest` uploaded as an artifact named `image_artifact_foo_latest`
  *      can be downloaded and loaded with ${downloadDir}/${packageName} i.e /tmp/foo_latest
  */
-exports.getDownloader = function (artifactDownloader, containerEngineName = "docker") {
+exports.getDownloader = function (artifactDownloader,
+                                  containerEngineName = "docker",
+				  dir = os.tmpdir()) {
     const containerEngine = getContainerEngine(containerEngineName);
     return async (image) => {
```

Comment:
```suggestion
    return async (image, tmpDownloadDirectory) => {
```

`download directory` feels like a parameter for the actual download function rather than a constructor 🤔 . What do you think ?

Reply:
> dir = os.tmpdir()

and does it work ? plz confirm. Nevermind if it works though 😄